README.md

```diff
@@ -178,7 +178,7 @@ Optionally:
 
 - add a `fargate-create.yml` ([example here](examples/fargate-create.yml)) to your template to drive custom configuration, prompting for defaults, etc. 
 
-For example (coming soon):
+For [example](https://github.com/turnerlabs/terraform-ecs-fargate-scheduled-task/blob/master/env/dev/fargate-create.yml):
```

Comment:
So you have an `example here` link to a local file and then link the word `example` to a GitHub blob in a different repo; seems like one too many example `fargate-create.yml`s?

Reply:
What am I missing?